master/internal/task/idle_timeout.go

```diff
@@ -39,7 +39,11 @@ func NewIdleTimeoutWatcher(name string, cfg *sproto.IdleTimeoutConfig) *IdleTime
 		IdleTimeoutConfig: *cfg,
 		Action: func(ctx *actor.Context) {
 			ctx.Log().Infof("killing %s due to inactivity", name)
-			ctx.Tell(ctx.Self(), sproto.TerminateAllocation)
+			ctx.Tell(ctx.Self(),
+				sproto.AllocationSignalWithReason{
+					AllocationSignal:    sproto.TerminateAllocation,
+					InformationalReason: "inactivity for more than 5s",
```

Comment:
This timeout is configurable and can be different than 5s

I think can be gotten from ```cfg.TimeoutDuration```

Reply:
Good point, I added that variable in to the new commit.